core/server/master/src/main/java/alluxio/master/file/FileSystemMasterClientServiceHandler.java

```diff
@@ -294,10 +296,19 @@ public void listStatusPartial(ListStatusPartialPRequest request,
   @Override
   public void mount(MountPRequest request, StreamObserver<MountPResponse> responseObserver) {
     RpcUtils.call(LOG, () -> {
-      mFileSystemMaster.mount(new AlluxioURI(request.getAlluxioPath()),
-          new AlluxioURI(request.getUfsPath()),
-          MountContext.create(request.getOptions().toBuilder())
-              .withTracker(new GrpcCallTracker(responseObserver)));
+      MountContext mountContext = MountContext.create(request.getOptions().toBuilder())
+          .withTracker(new GrpcCallTracker(responseObserver));
+      // By default, the mount execution process record is enabled
+      mountContext.getRecorder().setEnabled();
+      try {
+        mFileSystemMaster.mount(new AlluxioURI(request.getAlluxioPath()),
+            new AlluxioURI(request.getUfsPath()), mountContext);
+      } catch (Exception e) {
+        // if throw Exception, returns the record of mount execution process by exception message
+        Recorder recorder = mountContext.getRecorder();
+        recorder.recordIfEnabled(e.getMessage());
+        throw new AlluxioException(String.join("\n", recorder.takeRecords()), e);
+      }
```

Comment:
```suggestion
      } catch (Throwable t) {
        Recorder recorder = mountContext.getRecorder();
        recorder.recordIfEnabled(String.format("Mount failed: %s"), t);
        throw t;
      }
```

Reply:
We need to return the recorded message to the client through throw, So ` throw new AlluxioException(String.join("\n", recorder.takeRecords()), e);` is  necessary.